src/main/java/edu/ucsb/cs156/happiercows/controllers/UserCommonsController.java

```diff
@@ -87,6 +87,7 @@ public ResponseEntity<String> putUserCommonsByIdBuy(
         if(userCommons.getTotalWealth() >= commons.getCowPrice() ){
           userCommons.setTotalWealth(userCommons.getTotalWealth() - commons.getCowPrice());
           userCommons.setNumOfCows(userCommons.getNumOfCows() + 1);
+          userCommons.setTotalCowHealth(userCommons.getTotalCowHealth() + 100);
```

Comment:
This formula is incorrect.   Please see the detailed code review.

Also we should factor this out into it's own method so that we aren't repeating the formula again on line 115.